From: Noa Osherovich <noaos@mellanox.com>
Date: Wed, 18 Jan 2017 15:39:57 +0200
Subject: [PATCH 409/453] IB/core: Add scatter FCS flag to use in WQ creation
Patch-mainline: v4.11-rc1
Git-commit: 27b0df1175d543c46f63de6dccba98d0e5cf2276
References: FATE#321231 FATE#321473 FATE#322153 FATE#322149

Add a new creation flag to set the scatter FCS capability of a WQ.

Signed-off-by: Noa Osherovich <noaos@mellanox.com>
Reviewed-by: Majd Dibbiny <majd@mellanox.com>
Reviewed-by: Yishai Hadas <yishaih@mellanox.com>
Signed-off-by: Leon Romanovsky <leon@kernel.org>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/rdma/ib_verbs.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/rdma/ib_verbs.h b/include/rdma/ib_verbs.h
index 098e5a379e5f..0ace987f06e8 100644
--- a/include/rdma/ib_verbs.h
+++ b/include/rdma/ib_verbs.h
@@ -1531,6 +1531,7 @@ struct ib_wq {
 
 enum ib_wq_flags {
 	IB_WQ_FLAGS_CVLAN_STRIPPING	= 1 << 0,
+	IB_WQ_FLAGS_SCATTER_FCS		= 1 << 1,
 };
 
 struct ib_wq_init_attr {
-- 
1.8.5.6

